tests/optim/utils/test_images_utils.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+import unittest
+
+from captum.optim._utils.images import get_neuron_pos
+from tests.helpers.basic import BaseTest
+
+
+class TestGetNeuronPos(BaseTest):
+    def test_get_neuron_pos(self) -> None:
+        W, H = 128, 128
+        x, y = get_neuron_pos(H, W)
+
+        assert x == W // 2
+        assert y == H // 2
+
+        x, y = get_neuron_pos(H, W, 5, 5)

```

Comment:
I've added the new test cases!

Reply:
Thank you for addressing the comment. In general, I think that it is good to have each special case in a separate test case (in this case a separate method). In case one of those cases fails, it is easier to narrow it down to the specific problem.

Reply:
Okay, I've separated the test cases into separate methods!

Reply:
Thank you!